### ISSUE TITLE
Adding a column to an existing table

### ISSUE THREAD

**Issue by tschm** [ponyorm/pony#310]
Hello,
what's the best strategy to add an column to an existing table? Simply adding in the pony class and try via generate tables = True doesn't do the job.

**Comment by sashaaero** [comment]
Hello. That operation is called migration. We just released our migration tool, you can try it.
https://github.com/ponyorm/pony/tree/orm-migrations/pony/migrate

**Comment by kozlovsky** [comment]
No, the migration tool is not released yet, this is just a preview version. You can try it, but need to be aware that the generated tables in this branch have some differences from the current release 0.7.3.

**Comment by tschm** [comment]
I am not brave enough to try it. For now I modify the tables in SQL per hand and update the pony entity model...

**Comment by isaacmg** [comment]
Any update on this? A way to do migrations without having to open the SQL client would be very handy. Thanks.

**Comment by clintonm9** [comment]
Would be great to get the migration tool released!

**Comment by TomGoBravo** [comment]
If I had access I'd close this issue since https://github.com/ponyorm/pony/issues/16 will fix it.

**Comment by chris-allnutt** [comment]
This is the only think keeping me from ditching sqlalchemy, I have in the past used yoyo with success in python projects
